modules/client/src/client-common/interfaces/core.ts

```diff
@@ -21,6 +21,7 @@ export interface IClientWeb3Core {
     abi: ContractInterface,
   ) => Contract & T;
   getDaoFactoryAddress: () => string;
+  getPluginRepoRegistryAddress: () => string;
```

Comment:
Didn't we have a getter with a different name in the code from above?

Reply:
because I followed the example of `getDaoFactoryAddress()` in `Web3Module`

Reply:
Then I would suggest all following the same convention (getter if possible)